src/wam2layers/tracking/backtrack.py

```diff
@@ -70,31 +79,47 @@ def backtrack(
         )
         + region * precip * s_upper / (s_upper + s_lower)
     )
+    # TODO: find better way to deal with negative values
```

Comment:
Remove line 82 as we have addressed this now

Reply:
done